src/sql/util.py

```diff
@@ -250,3 +252,56 @@ def support_only_sql_alchemy_connection(command):
     """
     if Connection.is_custom_connection():
         raise exceptions.RuntimeError(f"{command} is not supported for a custom engine")
+
+
+def fetch_sql_with_pagination(
+    table, offset, n_rows, sort_column=None, sort_order=None
+) -> tuple():
```

Comment:
I don't think we need the `()`

```suggestion
) -> tuple:
```